cg/services/pacbio/metrics/models.py

```diff
@@ -37,3 +37,11 @@ class ControlMetrics(BaseModel):
     _validate_percent_mode_concordance_reads = field_validator(
         "percent_mode_concordance_reads", mode="before"
     )(fraction_to_percent)
+
+
+class ProductivityMetrics(BaseModel):
+    """Model for the loading metrics."""
+
+    p_0: int = Field(..., alias=LoadingAttributesIDs.P_0)
```

Comment:
Is this name ok or would you prefer `productivity_0`?

Reply:
I rather keep them the same as in the file without some translation. It will be easier to lookup in the manual on the website and find them in the files.

Reply:
we will use the property defined below